docs/yubikey_setup.rst

```diff
@@ -2,7 +2,7 @@ Using YubiKey with the Journalist Interface
 ===========================================
 
 This is a quick and dirty guide to using YubiKey for two-factor
-authentication on the Journalist Interface.
+authentication in the Journalist Interface.
```

Comment:
"on the Journalist Interface" is better

Reply:
I slightly favour in. However. "Add user page in the admin interface to add new users." is already used. Maybe change to "from" on both?